src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/AddInstanceV7.sql

```diff
@@ -0,0 +1,181 @@
+﻿/*************************************************************
+    Stored procedure for adding an instance.
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     AddInstanceV7
+--
+-- FIRST SCHEMA VERSION
+--     7
+--
+-- DESCRIPTION
+--     Adds a DICOM instance, now with partition.
+--
+-- PARAMETERS
+--     @partitionKey
+--         * The system identified of the data partition.
+--     @studyInstanceUid
+--         * The study instance UID.
+--     @seriesInstanceUid
+--         * The series instance UID.
+--     @sopInstanceUid
+--         * The SOP instance UID.
+--     @patientId
+--         * The Id of the patient.
+--     @patientName
+--         * The name of the patient.
+--     @referringPhysicianName
+--         * The referring physician name.
+--     @studyDate
+--         * The study date.
+--     @studyDescription
+--         * The study description.
+--     @accessionNumber
+--         * The accession number associated for the study.
+--     @modality
+--         * The modality associated for the series.
+--     @performedProcedureStepStartDate
+--         * The date when the procedure for the series was performed.
+--     @stringExtendedQueryTags
+--         * String extended query tag data
+--     @longExtendedQueryTags
+--         * Long extended query tag data
+--     @doubleExtendedQueryTags
+--         * Double extended query tag data
+--     @dateTimeExtendedQueryTags
+--         * DateTime extended query tag data
+--     @personNameExtendedQueryTags
+--         * PersonName extended query tag data
+-- RETURN VALUE
+--     The watermark (version).
+------------------------------------------------------------------------
+CREATE OR ALTER PROCEDURE dbo.AddInstanceV7
```

Comment:
Why do we need a new version? Since the signature is not changing? Basically tables and sprocs change together and client does not need to know anything the new prc will start pumping partitionKey.

Same may apply to other procedures.

Reply:
No, this procedure in turn calls `IIndexInstanceCoreV7` sproc, which uses new added partitionKey column .

Reply:
as discussed moved to older versions